plugins/auth-node/src/oauth/createOAuthRouteHandlers.ts

```diff
@@ -99,6 +99,9 @@ export function createOAuthRouteHandlers<TProfile>(
   const callbackUrl =
     config.getOptionalString('callbackUrl') ??
     `${baseUrl}/${providerId}/handler/frame`;
+  const cookieMaxAge = config.has('cookieMaxAge')
```

Comment:
I'm thinking we may want to keep this more oriented towards end outcome rather than the implementation. By that I mean a name such as `sessionMaxAge`, `sessionAgeLimit`, or `sessionDuration` could fit a bit better? One thing I specifically have in mind is the possibility of moving session storage to the backend in the future, at which point we'd be invalidating the DB entry rather than clearing the cookie. What do you think?

Reply:
I agree! I think `sessionDuration` would be the most intuitive in terms of the outcome. 

I'm thinking we can still communicate that this configuration is tied to the cookie lifespan in the documentation like this, WDYT?

```
- `sessionDuration` (optional): Lifespan of the refresh token cookie and granted scopes cookie
  issued during the authentication process.
```

Reply:
yep sounds good! :+1: